File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int temp1 = 0, temp2 = 0;

        for(int i = 0; i < s.length(); i++)
        {
            if(s.charAt(i) == 'p' || s.charAt(i) == 'P')
                temp1++;
            else if(s.charAt(i) == 'y' || s.charAt(i) == 'Y')
                temp2++;
        }

        return temp1 == temp2;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    let temp1 = 0, temp2 = 0;

    for(let i = 0; i < s.length; i++)
    {
        if(s[i] == 'p' || s[i] == 'P')
            temp1 += 1;
        else if(s[i] == 'y' || s[i] == 'Y')
            temp2++;
    }

    return temp1 == temp2;;
}